server/data/prisonApiClient.test.ts

```diff
@@ -41,6 +46,21 @@ describe('PrisonApiClient', () => {
     })
   })
 
+  describe('findOffenderBookingByOffenderNo', () => {
+    const offenderNo = 'A1234AA'
+    const offender: InmateDetail = {
+      offenceHistory: [offenceHistoryDetailFactory.build(), offenceHistoryDetailFactory.build({ mostSerious: true })],
+      offenderNo,
+    }
```

Comment:
Might we need an additional factory for this or will this be its only use?

Reply:
Yeah think it's nicer to have one.